web/src/components/Amenities/Amenities.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react'
```

Comment:
nice work here, a little bit of refactoring to abstract some of the UI into their own components